src/gen-address.js

```diff
@@ -0,0 +1,25 @@
+const faker = require('faker')
+
+const address = () => ({
+  street: address.streetAddress(),
+  city: address.city(),
+  state: address.state(),
```

Comment:
This should be `address.stateAbbr()`, we want to access the state through the name `state`, but we don't want the full name of the state.